e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/security/accesscontrol/deleteput/SimpleAccessControlRest.java

```diff
@@ -0,0 +1,150 @@
+package com.foo.rest.examples.spring.security.accesscontrol.deleteput;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.HashMap;
+
+/*
+Vulnerable application in which only authorized users can create or delete resources but
+any user can modify resources.
+
+This application was developed to satisfy the request:
+
+delete this file once added example
+
+each E2E will have its own subpackage here
+
+example, simple with only 2 endpoints: PUT and DELETE on
+
+/api/{x}
+
+ */
+
+@RestController
```

Comment:
also create some "manual" tests using RestAssured, to check it

Reply:
Hi Andrea,

Thanks for your comments. I removed the POST method. I am looking at RestAssured.

Reply:
Just as a note, I did not implement EMTests and Manual Tests yet, I will complete those after the application is good enough as a simple security test case.